009-crd-v1-roadmap.md

```diff
@@ -0,0 +1,118 @@
+
+
+# Roadmap to using the CRD v1 API
+
+This describes (roughly) the steps Strimzi users will need to take to get to using v1beta2 Strimzi APIs which, in turn, are using the v1 CRD API. 
+As such it talks about future versions of Strimz. We don't yet know for sure what versions the features described would land in, so we'll use the terms
+Strimzi X and Strimzi Z, where X and Z are unknown version numbers > 0.20 with X < Z. We've used Z rather than Y because it's not a given that Z is the version following X.
+The requirement is that what we’re calling Strimzi Z is released around the same time that Kubernetes 1.21 is.
+
+## Current situation
+
+Currently Strimzi defines numerous `CustomResourceDefinitions` using the `apiextensions.k8s.io/v1beta1` API. Those CRDs define the "Strimzi CRs", as `v1alpha1` and `v1beta1` APIS in the `kafka.strimzi.io` API group.
+
+Kubernetes 1.21 is expected to have removed CRD `apiextensions.k8s.io/v1beta1`. Its replacement, `apiextensions.k8s.io/v11`, has been available since Kubernetes 1.16.
```

Comment:
```suggestion
Kubernetes 1.22 is expected to have removed CRD `apiextensions.k8s.io/v1beta1`. Its replacement, `apiextensions.k8s.io/v11`, has been available since Kubernetes 1.16.
```